static/app/views/partials/org_tabs.html

```diff
@@ -0,0 +1,10 @@
+<div class="page-header">
+    <h3 class="text-center">{{activeOrg | orgNameFilter}} Spaces</h3>
+</div>
+<!-- Nav tabs -->
+<ul class="nav nav-tabs nav-justified">
```

Comment:
Changing this to `<ul class="nav nav-pills nav-justified">`  might help with the visual, the spaces sections are pills.